common/src/main/scala/rpgboss/player/InputMultiplexer.scala

```diff
@@ -65,6 +69,10 @@ object MyKeysEnum extends Enumeration {
   val Right = Value(3, "Right")
   val OK = Value(4, "OK")
   val Cancel = Value(5, "Cancel")
+  val A = Value(6, "A")
```

Comment:
The default keyboard mappings should be something more accessible. Maybe Ctrl, Shift, Alt, Enter?